lib/include/openamp/remoteproc_virtio.h

```diff
@@ -26,12 +26,15 @@ extern "C" {
 
 /* cache invalidation helpers for resource table */
 #ifdef VIRTIO_CACHED_RSC_TABLE
-#define RSC_TABLE_FLUSH(x, s)		metal_cache_flush(x, s)
-#define RSC_TABLE_INVALIDATE(x, s)	metal_cache_invalidate(x, s)
+#warning "VIRTIO_CACHED_RSC_TABLE is deprecated, please use VIRTIO_USE_CACHE"
+#endif
+#if defined(VIRTIO_CACHED_RSC_TABLE) || defined(VIRTIO_USE_CACHE)
```

Comment:
you update VIRTIO_USE_CACHE to VIRTIO_USE_DCACHE in `option.cmake` but not in the source files

Reply:
You're right. Thanks!
I fixed it!